src/gui/debuggingdialog.cpp

```diff
@@ -251,4 +674,78 @@ void DebuggingDialog::onSaveButtonTriggered(bool checked) {
     accept();
 }
 
+void DebuggingDialog::onLinkActivated(const QString &link) {
+    QString folderPath = "";
```

Comment:
Same, no need to initialize with empty string

Reply:
[e3e418d](https://github.com/Infomaniak/desktop-kDrive/pull/139/commits/e3e418d34a77a1d1be681396caaeeb199444068b)